salesagility/SuiteCRM
# AOR Reports Fixes

## Description



Fixes numerous issue with reports regarding the use of date fields with type "Value".
1. All operators for date fields with type "Value" return incorrectly
2. Reports only accept the format Y-m-d
3. PDF/CSV's do not respect date filtering
4. Report pagination returns a blank page when using date conditions
5.  Blank PDF/CSV when there are multiple parameters
6. 0 results shown when changing pages with two parameters
7. PDF/CSV have no data with date parameters
8. No option to use "Contains" operator on multi-select fields
9. "Contains" operator parameters fail to return

## Motivation and Context

Issue reference: #1624
Issue reference: #3686 
Issue reference: #2725
Issue reference: #3506
Issue reference: #3094
Issue reference: #3124
Issue reference: #3366
Issue reference: #4084
Issue reference: #2189

## How To Test This

1. Create reports using different date formats set in user profile.
2. Create reports with multiple parameters.
3. Check that report pagination returns correctly.
4. Check that exporting PDF/CSV works correctly with multiple parameters

## Types of changes


### Final checklist
